RFCs/backend/database/tutorial_related_tables/introduction.md

```diff
@@ -0,0 +1,71 @@
+|      tutorial translation title       |             `The title of a tutorial translation             |
+|      tutorial translation author      |             The author of a tutorial translation             |
+|     tutorial translation abstract     |            The abstract of a tutorial translation            |
+|     tutorial translation content      |          The main content of a tutorial translation          |
+| tutorial translation content markdown |    The markdown version of a tutorial translation content    |
+|   tutorial translation content html   |      The HTML version of a tutorial translation content      |
+
+## Graph
+
+|             Term             |                         Description                          |
+| :--------------------------: | :----------------------------------------------------------: |
+|         graph anchor         |              a collection of a graph's metadata              |
+|           graph ID           |                The unique UUID of the graph.                 |
+|          graph URL           |  The URL that users use to access the corresponding graph.   |
+|          graph name          |                    The name of the graph.                    |
+|          graph JSON          | The JSON description of the an actual graph. The format is in CYJS. |
+|          graph tag           |               Tag associated with this graph.                |
+|         graph maker          |                   Maker of the graph JSON.                   |
+|         graph source         |                   The source of the graph.                   |
+|        graph content         | The descrition of graphs in JSON and other format. This is different than the graph description. See below for more details. |
+|      graph description       | The table storing textual descriptionsof a graph, which states what the graph is about, etc. |
+|   graph description title    |              The title of the graph description              |
+|  graph description content   |              The textual description of a graph              |
+| tutorial of this graph (TBD) |            The tutorial that's using this graph.             |
+
+## Code
+
+|         Term         |                  Description                   |
+| :------------------: | :--------------------------------------------: |
+|       code ID        |          The unique UUID of the code           |
+|     code content     |             The actual texual code             |
+| tutorial of the code | The unique tutorial that talks about the code. |
+
+## Execution Result
+
+|        Term         |                   Description                    |
+| :-----------------: | :----------------------------------------------: |
+| Execution Result ID |      The unique UUID of a execution result.      |
+|  Execution Result   | The execution result of some code on some graph. |
```

Comment:
these two terms start with a capitalized letter, other terms don't.

Reply:
* Good question hmmm. Add an of <some language> after the content? 
* will fix that